pkg/policy/api/cidr.go

```diff
@@ -61,6 +62,15 @@ type CIDRRule struct {
 	Generated bool `json:"-"`
 }
 
+// String converts the CIDRRule into a human-readable string.
+func (r CIDRRule) String() string {
```

Comment:
I debated about whether to prepend an extra character like `@` to denote generated `CIDRRule`s, but avoided it for now since the only use will be in the test changes where it shouldn't make a difference whether the rule was generated or not.